tools/fpgabist/fpgabist

```diff
@@ -84,4 +84,7 @@ if __name__ == "__main__":
     parser = argparse.ArgumentParser()
     bist_common.global_arguments(parser)
     args = parser.parse_args()
+    if bist_common.check_commands():
+        sys.exit("Terminating BIST.")
```

Comment:
This should be moved to function. It is nice to have all features enclosed in function.